src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -65,7 +65,7 @@ def _load_file(path):
             import os
             content = read_file_content(os.path.expanduser(path), allow_binary=True)
 
-        return content[0:-1] if content and content[-1] == '\n' else content
+        return content.rstrip(os.linesep)
```

Comment:
?

Reply:
Ensures both Windows and Linux-style line endings are removed correctly (the old logic caused errors with TSV on Windows)